data/sql/updates/pending_db_world/rev_1741311298145347710.sql

```diff
@@ -0,0 +1,2 @@
+-- XOR PrevQuestID that was 499 Elixir of Suffering
+UPDATE `quest_template_addon` SET `PrevQuestID` = `PrevQuestID` ^ `PrevQuestID` WHERE `ID` = 501;
```

Comment:
just set it to `0` :D
```suggestion
UPDATE `quest_template_addon` SET `PrevQuestID` = 0 WHERE `ID` = 501;
```

Reply:
Haha ok, think last time I did simple SQL someone wanted it bitwise even if static change like this. I’ll just keep it simple next time